src/internal/disableStorage.ts

```diff
@@ -0,0 +1,36 @@
+export function disableStorage() {
+  if (!globalThis.localStorage) return
+
+  const noop = () => {
+    console.error("WARNING! Local storage is disabled when using Rune SDK.")
```

Comment:
FFTI: You use `console.error` but you prefix the msg with `WARNING` -  so that seems a bit inconsistent.
Also maybe you could improve the msg by adding that both local and session storages are disabled.

Reply:
I am notifying user not to use the storage, but it does not break the app, so I sent the warning message. But the yellow message is a bit easier to miss, especially if user is using some kind of game engine etc where getting some warnings in the console log is expected. I'll replace the text to ERROR since in the end we don't want the user to use the storage.

Reply:
[`ae4828c` (#46)](https://github.com/rune/rune-games-sdk/pull/46/commits/ae4828c47848de4f45534242dd4e9a5ef3f4797c)